src/backend/src/form-responses/dto/create-multi-form-responses.dto.ts

```diff
@@ -0,0 +1,5 @@
+type questionAnswerPair = [number, string];
+
+export class CreateMultiFormResponsesDto {
+  readonly questionAnswerPairs: questionAnswerPair[];
```

Comment:
I feel as though multiform response and form response should have similar formats, like if multiform is a questionAnswerPair[] then form response should be a single questionAnswerPair